ignite/cmd/testnet_multi_node.go

```diff
@@ -6,9 +6,10 @@ import (
 	"time"
 
 	tea "github.com/charmbracelet/bubbletea"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/spf13/cobra"
 
```

Comment:
```suggestion
```

Reply:
Nope it's the gci linter enforcing that.